From 2a1c578f7c446717a6a5eb2070190d42e6affcb6 Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Thu, 19 Jul 2018 14:20:26 -0700
Subject: [PATCH 6/9] interfaces: Do not add custom interfaces to VNDK

We shouldn't be adding or removing stuff from the VNDK, vendor
copies of custom HAL definitions need to go straight to vendor.

Change-Id: I228857be5dc259bfeb9b965e242473c7f9757d8d
---
 livedisplay/1.0/Android.bp | 3 ---
 power/1.0/Android.bp       | 3 ---
 stache/1.0/Android.bp      | 3 ---
 3 files changed, 9 deletions(-)

diff --git a/livedisplay/1.0/Android.bp b/livedisplay/1.0/Android.bp
index f24dbae..dd56301 100644
--- a/livedisplay/1.0/Android.bp
+++ b/livedisplay/1.0/Android.bp
@@ -46,9 +46,6 @@ cc_library {
     generated_headers: ["vendor.lineage.livedisplay@1.0_genc++_headers"],
     export_generated_headers: ["vendor.lineage.livedisplay@1.0_genc++_headers"],
     vendor_available: true,
-    vndk: {
-        enabled: true,
-    },
     shared_libs: [
         "libhidlbase",
         "libhidltransport",
diff --git a/power/1.0/Android.bp b/power/1.0/Android.bp
index 3741342..b94e80c 100644
--- a/power/1.0/Android.bp
+++ b/power/1.0/Android.bp
@@ -46,9 +46,6 @@ cc_library {
     generated_headers: ["vendor.lineage.power@1.0_genc++_headers"],
     export_generated_headers: ["vendor.lineage.power@1.0_genc++_headers"],
     vendor_available: true,
-    vndk: {
-        enabled: true,
-    },
     shared_libs: [
         "libhidlbase",
         "libhidltransport",
diff --git a/stache/1.0/Android.bp b/stache/1.0/Android.bp
index f1c9461..ce60a51 100644
--- a/stache/1.0/Android.bp
+++ b/stache/1.0/Android.bp
@@ -46,9 +46,6 @@ cc_library {
     generated_headers: ["vendor.lineage.stache@1.0_genc++_headers"],
     export_generated_headers: ["vendor.lineage.stache@1.0_genc++_headers"],
     vendor_available: true,
-    vndk: {
-        enabled: true,
-    },
     shared_libs: [
         "libhidlbase",
         "libhidltransport",
-- 
2.17.1

